mecanumDriveFieldCentric.java

```diff
@@ -41,6 +48,11 @@ public class MecanumDriveFieldCentric extends LinearOpMode {
 			private DcMotorEx MElbowRight;
 		//!SECTION - End arm motors
 
+		//SECTION - Claws
+			private CRServo clawLeft;
+			private CRServo clawRight;
+		//!SECTION - End arm motors
```

Comment:
Change to "End claw setup" or alike

Reply:
In all honesty, all of these comments for sections are extraneous, and likely just contribute to issues with readability.

The constants should be put in their own file that can be imported, so there is one place for configuration, reducing the muddiness of commits in the long run.

That having been said -- I think this is a nitpick, and not really a blocker for code like this.

Reply:
Agreed